R/approach.R

```diff
@@ -43,31 +39,13 @@ setup_approach.combined <- function(internal, ...) {
 
 #' Generate data used for predictions and Monte Carlo integration
 #'
-#' @note
```

Comment:
Also, in the future: put stuff like this in a comment here on github instead.

Reply:
Noted. Makes more sense to put it here than in the code...